tldw_Server_API/app/core/DB_Management/Jobs_Repository.py

```
@@ -0,0 +1,240 @@
+
+    @classmethod
+    def for_sqlite(cls, db_path: Path) -> "JobsRepository":
+        """Create a repository configured for the SQLite jobs database."""
+        return cls(backend="sqlite", db_path=db_path)
+
+    @classmethod
+    def for_postgres(cls, db_url: str) -> "JobsRepository":
+        """Create a repository configured for the PostgreSQL jobs database."""
+        return cls(backend="postgres", db_url=db_url)
+
+    def _connect(self) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_connect`

(ANN401)